xcell/mappers/mapper_DESY1gc.py

```diff
@@ -7,6 +7,8 @@
 
 
 class MapperDESY1gc(MapperBase):
+    map_name = 'DESY1gc'
+
```

Comment:
Don't you need to update this name depending on the bin?

Reply:
See reply in wl mapper